src/diff/props.js

```diff
@@ -35,6 +35,7 @@ let CAMEL_REG = /-?(?=[A-Z])/g;
 function setProperty(dom, name, value, oldValue, isSvg) {
 	let v;
 	if (name==='class' || name==='className') name = isSvg ? 'class' : 'className';
+	const ns = isSvg && (name !== (name = name.replace(/^xlink:?/, '')));
```

Comment:
Do we have to restrict this to SVG elements? I think setting namespaced attributes on normal dom elements (while not as common) is still valid, so perhaps we can save some bytes by removing the `isSvg` check?

Reply:
Should be able to leave this enabled for any element. I believe it was guarded by isSvg in Preact 8 just to avoid the cost of doing a comparison and `.replace()` call in the more common non-svg case.

Reply:
@JoviDeCroock I'm wondering:  is it possible to move this line (38) down into the attribute set/remove clauses on 88 and 91?  It's duplication, but it would mean the vast majority of setProperty() calls never have to run this code.

Reply:
>  avoid the cost of doing a comparison and .replace() call in the more common non-svg case

Oh, true. That make sense :)